Source/WebKit/UIProcess/API/glib/WebKitFaviconDatabase.h.in

```diff
@@ -36,6 +36,32 @@
 
 G_BEGIN_DECLS
 
+#if PLATFORM(WPE) && ENABLE(2022_GLIB_API)
+#define WEBKIT_TYPE_FAVICON (webkit_favicon_get_type())
+typedef struct _WebKitFavicon WebKitFavicon;
```

Comment:
I suppose this could be called `WPEImage` or something similar if we want to reuse it in more places.

Reply:
@carlosgcampos Any thoughts on this?

Reply:
Actually, I am going to call it `WebKitImage` and make it a `GObject` subclass, that way we will be able to make it implement the `GIcon` interface (and maybe `GLoadableIcon` as well). Plus, then we could reuse it to implement `webkit_web_view_get_snapshot()` for WPE in the future.